src/test/java/com/commercetools/emailprocessor/model/TenantConfigurationTest.java

```diff
@@ -0,0 +1,91 @@
+package com.commercetools.emailprocessor.model;
+
+import io.sphere.sdk.client.SphereApiConfig;
+import io.sphere.sdk.client.SphereClient;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import static org.junit.Assert.*;
+
+public class TenantConfigurationTest {
```

Comment:
I think you don't need to test getters and setters.

Reply:
1683e6a